DataProcessing/datax-host/src/main/scala/datax/fs/HadoopClient.scala

```diff
@@ -127,8 +127,15 @@ object HadoopClient {
       case Some(value)=>
         logger.warn(s"Retrieved key for storage account '$sa' with secretid:'$secretId'")
         setStorageAccountKey(sa, value)
-      case None =>
-        logger.warn(s"Failed to find key for storage account '$sa' with secretid:'$secretId'")
+      case None =>	    
+        val databricksSecretId = s"secretscope://$vaultName/${ProductConstant.ProductRoot}-sa-$sa"
```

Comment:
can you add a comment to detail what and why we are doing this?  Also, is this a possibility to add a unit test here?

Reply:
Added comment. Adding unit tests will be done as a separate work item